ClockDisplay.java

```diff
@@ -24,7 +24,7 @@
      */
     public ClockDisplay()
     {
-        hours = new NumberDisplay(24);
+        hours = new NumberDisplay(13);
```

Comment:
this allows for 13 hour range 0..12 - the number display should still be from 0..11